recipe/meta.yaml

```diff
@@ -1,63 +1,71 @@
-{% set version = "0.18.0" %}
+{% set version = "0.21.1" %}
 
 package:
   name: cartopy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/C/Cartopy/Cartopy-{{ version }}.tar.gz
-  sha256: 7ffa317e8f8011e0d965a3ef1179e57a049f77019867ed677d49dcc5c0744434
-  patches:
-    - port310.patch
+  sha256: 89d5649712c8582231c6e11825a04c85f6f0cee94dbb89e4db23eabca1cc250a
 
 build:
-  skip: True  # [win and vc<14]
-  error_overlinking: true
-  number: 2
+  # s390x is missing geos and this package is not in the SoW
+  skip: True  # [(win and vc<14) or s390x or py<38]
+  number: 0
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch  # [not win]
-    - m2-patch  # [win]
   host:
     - python
     - pip
-    - cython
-    - numpy
-    - msinttypes  # [win and vc<14]
-    # proj and geos are needed as dependencies even though not listed upstream
-    - proj <8.0.0  # newer versions removed required proj_api.h
+    - setuptools
+    - setuptools_scm
+    - wheel
+    - cython 0.29
+    - numpy   1.19  # [py<310]
+    - numpy   1.21  # [py==310]
+    - numpy   1.23  # [py>=311]
     - geos
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - shapely >=1.5.6
-    - pyshp >=1.1.4
-    - six >=1.3.0
-    # proj and geos are needed as dependencies even though not listed upstream
-    - proj <8.0.0  # newer versions removed required proj_api.h
-    - geos
+    - shapely >=1.6.4
+    - pyshp >=2.1
+    - geos >=3.7.2
     # including matplotlib-base even though it's optional upstream
-    - matplotlib-base >=1.5.1
-    - scipy >=0.10
+    - matplotlib-base >=3.1
+    - scipy >=1.3.1
+    - pyproj >=3.0.0
 
 test:
-  imports:
```

Comment:
It's best to keep the imports imho.

Reply:
The same imports are tested in `run_test.py`  It's my understanding those imports aren't even being tested when `run_test.py` is present.

Reply:
Or was it if you have a run_test.sh that the commands part is not executed?
From conda build code it looks to me that this would still be executed.
Any case, run_test.py could also have `import cartopy` there.

Reply:
`import cartopy` is the first line in `run_test.py`

We might want file a bug in the linter if this is the case.

Reply:
Would need some digging.

I only see `import cartopy.crs`.
I would replace by

```
import cartopy
import cartopy.mpl.geoaxes
import cartopy.crs
```

Reply:
Sorry, I was looking at the wrong branch. All is good.